File: ERP/wwwroot/js/modelos/almacen/AConsumoEconomato.js
class AConsumoEconomato {
    constructor() {
        this.idconsumoeconomato = 0;
        this.numdocumento = "";
        this.motivo = "";
        this.estado = "HABILITADO";
    }
}

class AConsumoEconomatoDetalle {
    constructor() {
        this.idconsumoeconomatodetalle = 0;
        this.idconsumoeconomato = 0;
        this.idproducto = 0;
        this.idstock = 0;
        this.cantidad = 0;
        this.estado = "HABILITADO";
    }
}

File: ENTIDADES/Almacen/AConsumoEconomato.cs
using ENTIDADES;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ENTIDADES.Almacen
{
    [Table("ConsumoEconomato", Schema = "Almacen")]
    public class AConsumoEconomato : AuditoriaColumna
    {
        [Key]
        public int idconsumoeconomato { get; set; }
        public string numdocumento { get; set; }
        public string motivo { get; set; }
        public string estado { get; set; }
        public int? suc_codigo { get; set; }
        [NotMapped]
        public string jsondetalle { get; set; }
    }
}
